src/phoenix/trace/filter.py

```diff
@@ -79,8 +79,17 @@ def _allowed_replacements() -> Iterator[Tuple[str, ast.expr]]:
         )
     }.items():
         yield source_segment, ast_replacement
-        yield "attributes." + source_segment, ast_replacement
-        yield "span.attributes." + source_segment, ast_replacement
```

Comment:
I think we will want this back once I add the column selector. What makes it clutter? I thought it was good to have! particularly when we have evaluation attributes.

Reply:
I've since added it back, because I was ambivalent myself.